cdap-watchdog/src/main/java/co/cask/cdap/logging/framework/LogPipelineLoader.java

```diff
@@ -109,7 +107,6 @@ public void validate() throws InvalidPipelineException {
 
     Map<String, LogPipelineSpecification<T>> result = new HashMap<>();
     Set<String> checkpointPrefixes = new TreeSet<>();
-    checkpointPrefixes.add(RESERVED_CHECKPOINT_PREFIX);
```

Comment:
This is because it was used to differentiate between metadata and checkpoints as they were shared table. Now that we have separated them out. It is not reserved checkpoint prefix any more.